lib/suspenders/app_builder.rb

```diff
@@ -196,13 +196,11 @@ def configure_i18n_for_missing_translations
     end
 
     def configure_i18n_tasks
-      run "cp $(i18n-tasks gem-path)/templates/rspec/i18n_spec.rb spec/"
       copy_file "config_i18n_tasks.yml", "config/i18n-tasks.yml"
     end
 
     def configure_background_jobs_for_rspec
       copy_file 'background_jobs_rspec.rb', 'spec/support/background_jobs.rb'
-      run 'rails g delayed_job:active_record'
```

Comment:
I don't think this PR is shippable with these lines completely removed, but with lines in the Gemfile merely commented out. It doesn't seem very helpful if you wanted to use Refills, for instance (you'd have to remember to un-comment out the line in the Gemfile, but then run `rails generate refills:import flashes`).